CircleCi nightly regression nightly-simulate-network-outage failed

**Summary of the defect**
at run-network-sim step.

**How to reproduce (if possible)**
https://app.circleci.com/pipelines/github/hashgraph/hedera-services/2934/workflows/1dab4326-35bd-44dd-bcfc-0ba1cb7f2122/jobs/30273

**Service logs (if applicable)**
```
...
2020-08-31 12:42:34.049 INFO   151  RunLoadTest - Thread thread30 ops 157 current TPS 15.60481
2020-08-31 12:42:36.903 WARN   266  HapiApiSpec - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.1114 <- +1, 0.0.1113 -> -1]} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<OK>
	at org.junit.Assert.fail(Assert.java:88) ~[junit-4.12.jar:4.12]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[junit-4.12.jar:4.12]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[junit-4.12.jar:4.12]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:243) ~[classes/:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:323) ~[classes/:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:264) ~[classes/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-08-31 12:42:36.903 WARN   266  HapiApiSpec - 'RunCryptoTransfers' - HapiCryptoTransfer{sigs=1, payer=sender, node=0.0.3, transfers=[0.0.1114 <- +1, 0.0.1113 -> -1]} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<OK>
	at org.junit.Assert.fail(Assert.java:88) ~[junit-4.12.jar:4.12]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[junit-4.12.jar:4.12]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[junit-4.12.jar:4.12]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:243) ~[classes/:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:323) ~[classes/:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[classes/:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:264) ~[classes/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
...
```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1079,6 +1079,13 @@ workflows:
           requires:
             - start-singlejob-testnet
   nightly-simulate-network-outage:
+    triggers:
+        - schedule:
+            cron: "0 0,12 * * *"
+            filters:
+              branches:
+                only:
+                  - master
     jobs:
       - fast-build-artifact:
           context: Slack

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -3131,7 +3131,7 @@ jobs:
             '/repo/.circleci/scripts/firewall_create_rules.sh'
       - run-eet-suites:
           dsl-args: "CryptoTransferLoadTest status.preResolve.pause.ms=8000"
-          ci-properties-map: mins=50
+          ci-properties-map: mins=50,tps=10,threads=3
       - run:
           name: call script to remove firewall rules
           command: |

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1079,13 +1079,6 @@ workflows:
           requires:
             - start-singlejob-testnet
   nightly-simulate-network-outage:
-    triggers:
-        - schedule:
-            cron: "0 0,12 * * *"
-            filters:
-              branches:
-                only:
-                  - master
     jobs:
       - fast-build-artifact:
           context: Slack

